plugins/out_s3/s3.c

```diff
@@ -1651,6 +1651,14 @@ static struct flb_config_map config_map[] = {
      "Send the Content-MD5 header with object uploads, as is required when Object Lock is enabled"
     },
 
+    {
+     FLB_CONFIG_MAP_BOOL, "static_file_path", "false",
+     0, FLB_TRUE, offsetof(struct flb_s3, static_file_path),
+     "Disables behavior where UUID string is automatically appended to end of S3 key name when "
```

Comment:
I think we should clarify the behavior when ```$UUID``` or ```$TAG``` or date/time specific formatters (```%Y/%m/%d/%H/%M/%S```) are specified and ```static_file_path``` is set to ```true```.